yarn.lock

```diff
@@ -18690,7 +18689,7 @@ vm-browserify@~0.0.1:
   dependencies:
     indexof "0.0.1"
 
-vm2@^3.9.17, vm2@^3.9.18:
+vm2@^3.9.18:
```

Comment:
why do we still need this?

Reply:
Probably from not removing node_modules during a local install first. I'll run that and see